artview/components/select_region.py

```diff
@@ -40,35 +41,38 @@ class SelectRegion(Component):
     '''
 
     Vpoints = None  #: see :ref:`shared_variable`
+    VplotAxes = None  #: see :ref:`shared_variable`
+    VpathInteriorFunc = None  #: see :ref:`shared_variable`
     Vgatefilter = None  #: see :ref:`shared_variable`
     Vradar = None  #: see :ref:`shared_variable`
+    Vfield = None  #: see :ref:`shared_variable`
 
     @classmethod
     def guiStart(self, parent=None):
-        args, independent = _SelectRegionStart().startDisplay()
-        return self(**args), independent
+        #args, independent = _SelectRegionStart().startDisplay()
+        kwargs, independent = \
+            common._SimplePluginStart("AccessTerminal").startDisplay()
+        kwargs['parent'] = parent
+        return self(**kwargs), independent
 
-    def __init__(self, display, name="SelectRegion", parent=None):
+
+    def __init__(self, VplotAxes=None, VpathInteriorFunc=None, Vfield=None,
+                 name="SelectRegion", parent=None):
         '''
```

Comment:
By adding the ```pathInteriorFunc```, etc here, you are now able to not have SelectRegion dependent upon an individual Display, correct?